.env

```diff
@@ -0,0 +1,6 @@
+PG_USER=topology-postgres-dev-1
```

Comment:
I should have noted this but whenever you add secrets to a repo do a quick check that they are included in the .gitignore file and won't be commited for the public to see.